fe/fe-core/src/main/cup/sql_parser.cup

```diff
@@ -4951,24 +4951,44 @@ window_boundary ::=
 arithmetic_expr ::=
   expr:e1 STAR expr:e2
   {: RESULT = new ArithmeticExpr(ArithmeticExpr.Operator.MULTIPLY, e1, e2); :}
+  | KW_MULTIPLY LPAREN expr:e1 COMMA expr:e2 RPAREN
```

Comment:
```functioname(param1, param2)``` is belongs to FunctionCallExpr .
If each functionname has to add grammar, the grammar file will not be able to be put down long ago